doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -488,6 +488,18 @@ Debug libraries
 Modem libraries
 ---------------
 
+* :ref:`lte_lc_readme` library:
+
```

Comment:
Aren't you also adding the new Kconfig `LTE_CONNECTIVITY`?

Reply:
It doesn't have an rst yet. It will be added during the doc improvement phase. Then we can add the reference here.